docs/spec/relayer/Chain.tla

```diff
@@ -156,6 +299,8 @@ Init ==
 Next ==
     \/ AdvanceChain 
     \/ HandleIncomingDatagrams
+    \/ SendPacket
+    \/ AcknowledgePacket
```

Comment:
Should we add timeout handling here? Then it seems complete.

Reply:
Also, what about channel closing? Is it also triggered "externally"?

Reply:
channel closing is triggered by the application so yes the trigger is "external"